### PR TITLE
remove false comments from RV32 permission encoding table

### PR DESCRIPTION
**Pull request by tariqkurd-repo**
quadrants 2 and 3 has a false comment copy/pasted from quadrant 0 most likely

### CODE REVIEW

**Review by andresag01 — APPROVED** [review]

File: src/cap-description.adoc [hunk 1]
```diff
@@ -194,13 +194,13 @@ Quadrant 1 encodes permissions for executable capabilities and the <<m_bit>>.
 | 4-5   | ✔ | ✔ | ✔ | ✔ | ✔ |     | Mode^1^  | Execute + Data & Cap RW
 | 6-7   | ✔ | ✔ |   |   | ✔ |     | Mode^1^  | Execute + Data RW
 9+| *Quadrant 2: Restricted capability data read/write*
-9+| bit[2] - write. R and C implicitly granted, LM dependent on W permission.
```
Comment (arichardson) [inline src/cap-description.adoc:197]: This is also true, see #355 although right now all of the writable encodings are reserved.

**Review by arichardson — COMMENTED** [review]
I guess we can remove this from quandrant 2 while there are no defined encodings where this matters.

File: src/cap-description.adoc [hunk 2]
```diff
@@ -194,13 +194,13 @@ Quadrant 1 encodes permissions for executable capabilities and the <<m_bit>>.
...
+9+| R and C implicitly granted, LM dependent on W permission.
 9+| _Reserved bits for future extensions must be 1 so they are implicitly granted_
 | 0-2   8+| reserved
 | 3       | ✔ |   | ✔ |   |   |     | N/A | Data & Cap RO (no LM)
 | 4-7   8+| reserved
 9+| *Quadrant 3: Capability data read/write*
-9+| [2] - write. R and C implicitly granted.
```
[the remark this answers is not in the document]
Comment (tariqkurd-repo) [reply src/cap-description.adoc:203]: ok, reinstated

File: src/cap-description.adoc (hunk 1 above)
Comment (tariqkurd-repo) [reply src/cap-description.adoc:197]: ok - I added a comment about that

**Review by arichardson — APPROVED** [review]

**Pull request merged**